settings/provisioner/virsh/virsh.spec

```diff
@@ -8,28 +8,39 @@ subparsers:
                   host:
                       type: str
                       help: Address/FQDN of the BM hypervisor
+                      required: yes
                   ssh-user:
                       type: str
                       help: User to SSH to the host with
+                      default: root
```

Comment:
Please add '(default: __DEFAULT__)' to the option's help wherever 'default' is defined, so the default value will be printed in the help screen.